### PR TITLE
Feature Request: Added abbreviated endpoint support for AP3 Jakarta

### PR DESCRIPTION
**Pull request by qgervacio**
### Description

This feature is to add abbreviated endpoint support for AP3 Jakarta needed for our migration.

### Acceptance tests
- [ ] Have you added an acceptance test for the functionality being added?
- [ ] Have you run the acceptance tests on this branch?

Output from acceptance testing:

N/A
```

## Release 1.18.8

New Features:

* added abbreviated endpoint for `AP3`.
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -687,4 +687,10 @@ New Features:
 New Features:
 
 ### data-source/coralogix_webhook
-* Added support for searching by webhook name.
\ No newline at end of file
+* Added support for searching by webhook name.
+
+## Release 1.18.8
```
Comment (celaus) [inline CHANGELOG.md:692]: Technically it will be 1.18.12 :)

File: docs/index.md [hunk 2]
```diff
@@ -94,5 +95,5 @@ For more information on the endpoints, check out [https://coralogix.com/docs/cor
   environment. environment variable 'CORALOGIX_API_KEY' can be defined instead.
 - `domain` (String) The Coralogix domain. Conflict With 'env'. environment variable 'CORALOGIX_DOMAIN' can be defined
   instead.
-- `env` (String) The Coralogix API environment. can be one of ["USA1" "USA2" "APAC1" "APAC2" "EUROPE1" "EUROPE2" "US1" "US2" "AP1" "AP2" "EU1" "EU2"]. environment
+- `env` (String) The Coralogix API environment. can be one of ["USA1" "USA2" "APAC1" "APAC2" "EUROPE1" "EUROPE2" "US1" "US2" "AP1" "AP2" "AP3" "EU1" "EU2"]. environment
```
Comment (celaus) [inline docs/index.md:98]: APAC3 should also be possible then?

File: CHANGELOG.md (hunk 1 above)
Comment (qgervacio) [reply CHANGELOG.md:692]: hmm. looks like the entry in `CHANGELOG.md` is missed for 1.18.8-11?

Comment (celaus) [reply CHANGELOG.md:692]: 😓

**Comment by qgervacio** [comment]
Hi @celaus. i'm trying the suggestion of our TAM to use env `CORALOGIX_ENV: AP3` and `CORALOGIX_DOMAIN: ap3.coralogix.com` but it seems there's a hard validation somewhere?

result from gitlab job. i have also tried `CORALOGIX_ENV=APAC3`

```
$ env | grep CORALOGIX
CORALOGIX_ENV=AP3
CORALOGIX_DOMAIN=ap3.coralogix.com
$ terraform plan -out tfplan
╷
│ Error: expected env to be one of [EUROPE1 EUROPE2 USA1 APAC1 APAC2], got AP3
│ 
│   with provider["registry.terraform.io/coralogix/coralogix"],
│   on providers.tf line 50, in provider "coralogix":
│   50: provider "coralogix" {
│ 
╵
```

**Comment by qgervacio** [comment]
Hi @celaus 

When using `CORALOGIX_ENV: AP3` and  `CORALOGIX_DOMAIN: ap3.coralogix.com`

```
│ Error: The Coralogix env must be one of ["AP1" "AP2" "EUROPE1" "EU1" "EUROPE2" "EU2" "US1" "APAC1" "USA1" "USA2" "US2" "APAC2"]
│ 
│   with provider["registry.terraform.io/coralogix/coralogix"],
│   on providers.tf line 50, in provider "coralogix":
│   50: provider "coralogix" {
│ 
╵
```

Can we expedite the change please?

**Comment by celaus** [comment]
I added the changes to #285

**Pull request closed without merging**